test/functional/example_test.py

```diff
@@ -164,13 +164,13 @@ def run_test(self):
         self.tip = int(self.nodes[0].getbestblockhash(), 16)
         self.block_time = self.nodes[0].getblock(self.nodes[0].getbestblockhash())['time'] + 1
 
-        height = 1
+        height = self.nodes[0].getblockcount()
```

Comment:
Actually the ` + 1` should be suffixed here (similar to block_time above?)

And then remove the change further down?

```suggestion
        height = self.nodes[0].getblockcount() + 1
```

Reply:
2+10=12, and we're waiting for block 11, I'd have to change that too.

Reply:
Ok, fine.